.github/workflows/update-ccv2.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  workflow_dispatch:
+    inputs:
+      ccv2_branch:
+        description: Branch on ccv2 to update
+        default: spa_p4
+        required: true
+      source_branch_to_deploy:
+        description: source branch to deploy on ccv2
+        default: develop
+        required: true
+
+name: Update ccv2 repo with unreleased Spartacus
+
+env:
+  DEFAULT_BRANCH_TO_DEPLOY: 'develop'
+
+jobs:
+  deploy_to_ccv2: 
+    name: Updating storefront of a ccv2 environment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.source_branch_to_deploy || env.DEFAULT_BRANCH_TO_DEPLOY }}
+          fetch-depth: 0
+      - name: Get commit hash of the branch to deploy
+        id: branch-to-deploy-commit-id
+        run: |
+          echo "::set-output name=commit_hash::$(git rev-parse ${{ github.event.inputs.source_branch_to_deploy || env.DEFAULT_BRANCH_TO_DEPLOY }})"
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: | 
+            node_modules
+          key: nodemodules-${{ steps.branch-to-deploy-commit-id.outputs.commit_hash }}
+          restore-keys: nodemodules-${{ steps.branch-to-deploy-commit-id.outputs.commit_hash }}
+      - name: Install angular CLI
+        run: npm install -g @angular/cli@v13-lts
```

Comment:
Is this on angular v13 on purpose?

Reply:
I can bump to 14, but there's no issue. It's just getting an ng version on the machine.

But we still just clone spartacus and the repo is on ng 14 if it's develop / release/4.3.x


Should we? If yes, then we need to bump the other pipeline files as it's on v13-lts. I can open the PR up for right now

Reply:
bam https://github.com/SAP/spartacus/pull/16214